tests/functionnal/Ticket.php

```diff
@@ -1020,33 +1020,33 @@ public function testFormTech() {
          $location = true
       );
 
-      //drop update ticket right from tech profile
+      //drop UPDATE ticket right from tech profile (still with OWN)
```

Comment:
Maybe a new test with tech without update and without own should be a good idea.